modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -426,3 +381,80 @@ const NewOverflowList = () => {
   );
 };
 ```
+
+## General Changes
+
+### Remove Default Imports
+
+As part of some cleanup, we've removed any default imports that where lingering. This change matches
+more closely what we've been doing with new components and systems. As part of this change, the
+components listed below have moved from default imports to named imports.
+
+#### Fonts
+
+- fonts
+
+#### Main Package Components
+
+- Toast
+- Canvas
+- Tooltip
+- TextArea
+- Avatar
+- CountBadge
+- Checkbox
+- FormField
+- LoadingDots
+- Radio
+- SegmentedControl
+- Select
+- SidePanel
+- StatusIndicator
+- Switch
+- Table
+- TextInput
+
+#### Labs Components
+
+- Combobox
+- Drawer
+
+#### Preview Components
+
+- Breadcrumbs
+- ColorPicker
+- Menu
+- Select
+- SidePanel
+
+🤖 The codemod will update imports to be named imports
+
+### Reorganize Style Props
+
+Some style props have reorganized and the implementation has been refactored, but the functionality will remain the same. This will improve the style prop documentation and allow Canvas Kit to extend style props further in the future. The style props `background` and `backgroundImage` have moved from the `color` style prop function to a new style prop function, `background`.
```

Comment:
```suggestion
Some style props have been reorganized and the implementation has been refactored, but the functionality will remain the same. This will improve the style prop documentation and allow us (Canvas Kit) to extend style props further in the future. The style props `background` and `backgroundImage` have moved from the `color` style prop function to a new style prop function, `background`.
```

Reply:
Feel free to disregard this, but this makes it seem more personal?